cypress/e2e/data-import/e2e/match-on-pol-and-update-instance.cy.js

```diff
@@ -23,11 +23,12 @@ import InventoryInstance from '../../../support/fragments/inventory/inventoryIns
 import InventoryViewSource from '../../../support/fragments/inventory/inventoryViewSource';
 import Users from '../../../support/fragments/users/users';
 import FileManager from '../../../support/utils/fileManager';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
 
 describe('ui-data-import', () => {
   let user = null;
   let orderNumber;
-  let instanceHrid;
+  const itemBarcode = '242451241241';
```

Comment:
why it is not a random number?) I assume better to use here is getRandomPostfix()